internal/meta/meta_rg.go

```diff
@@ -117,13 +117,16 @@ func (meta MetaResourceGroup) queryResourceSet(ctx context.Context, rg string) (
 		rl = append(rl, res)
 	}
 
-	// Especially, adding the resoruce group itself to the resource set
-	rl = append(rl,
-		resourceset.AzureResource{Id: &armid.ResourceGroup{
-			SubscriptionId: meta.subscriptionId,
-			Name:           meta.resourceGroup,
-		}},
-	)
+	// Especially, if there is no resource within the resource group, the azlist will return an empty list.
```

Comment:
Actually, this is not what I happened, imho. The resource group was added into the list, that was not empty: [the problem](https://github.com/Azure/aztfexport/issues/573) was that the role assignment was missing.